class/defaults.yml

```diff
@@ -78,6 +82,26 @@ parameters:
             memory: 300Mi
         kustomize_input:
           namespace: ${appcat:slos:namespace}
+      alerting:
+        labels:
+          syn_component: "appcat"
+          slo: "true"
+        page_labels:
+          severity: "critical"
+        ticket_labels:
+          severity: "warning"
+      vshn:
+        postgres:
+          uptime:
+            objective: 99.9
+            alerting:
+              page_alert:
+                # This should reduce non actionable alerts because of single instance restarts.
+                # The page alert looks (ammong other things) at the burn rate over the last 5min.
+                # If the alert is pending for more than 5m this indicates a real problem.
+                for: 6m
```

Comment:
We said a restart usually is about 1 min, any reasons we have 6m here?

Reply:
Because the alert uses the rate over 5 minutes. So a ~30s might trigger the alert for 5min. Not completely happy with this solution, but I don't have a better idea right now

Reply:
we cannot diminish the rate to say 2 min? It's not feasible from your point of view?

Reply:
No that's defined by Sloth and just how Google's SLO alerting framework should work.